docs/content/preview/api/ysql/the-sql-language/statements/ddl_drop_index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: DROP INDEX statement [YSQL]
+headerTitle: DROP INDEX
+linkTitle: DROP INDEX
+description: Use the DROP INDEX statement to remove one or more indexes from the database.
+menu:
+  preview:
+    identifier: ddl_drop_index
+    parent: statements
+aliases:
+  - /preview/api/ysql/commands/ddl_drop_index/
+type: docs
+---
+
+## Synopsis
+
+Use the `DROP INDEX` statement to remove one or more indexes from the database.
+
+## Syntax
+
+<ul class="nav nav-tabs nav-tabs-yb">
+  <li >
+    <a href="#grammar" class="nav-link active" id="grammar-tab" data-toggle="tab" role="tab" aria-controls="grammar" aria-selected="true">
+      <img src="/icons/file-lines.svg" alt="Grammar Icon">
+      Grammar
+    </a>
+  </li>
+  <li>
+    <a href="#diagram" class="nav-link" id="diagram-tab" data-toggle="tab" role="tab" aria-controls="diagram" aria-selected="false">
+      <img src="/icons/diagram.svg" alt="Diagram Icon">
+      Diagram
+    </a>
+  </li>
+</ul>
+
+<div class="tab-content">
+  <div id="grammar" class="tab-pane fade show active" role="tabpanel" aria-labelledby="grammar-tab">
+  {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/drop_index.grammar.md" %}}
+  </div>
+  <div id="diagram" class="tab-pane fade" role="tabpanel" aria-labelledby="diagram-tab">
+  {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/drop_index.diagram.md" %}}
+  </div>
+</div>
+
+## Semantics
+
+#### *if_exists*
+
+Under normal operation, an error is raised if the index does not exist.  Adding `IF EXISTS` will quietly ignore any non-existent indexes specified.
+
+#### *index_name*
+
+Specify the name of the index to be dropped. Objects associated with the index will be invalidated after the `DROP INDEX` statement is completed.
+
+#### RESTRICT / CASCADE
+
+`RESTRICT` (the default) will not drop the index if any objects depend on it.
+
+`CASCADE` will drop any objects that transitively depend on the index.
+
+## Example
+
+Create a table with an index:
+
+```plpgsql
+CREATE TABLE t1(id BIGSERIAL PRIMARY KEY, v TEXT);
+CREATE INDEX i1 ON t1(v);
+```
+
+Verify the index was created:
+
+```sql
+\d t1
+```
+
+```output
+                            Table "public.t1"
+ Column |  Type  | Collation | Nullable |            Default             
+--------+--------+-----------+----------+--------------------------------
+ id     | bigint |           | not null | nextval('t1_id_seq'::regclass)
+ v      | text   |           |          | 
+Indexes:
+    "t1_pkey" PRIMARY KEY, lsm (id HASH)
+    "i1" lsm (v HASH)
+```
+
+Drop the index:
+
+```sql
+DROP INDEX i1;
+```
+
+Use the `\d t1` meta-command to verify that the index no longer exists.
```

Comment:
Why not show the output as well?

Reply:
Fixed in https://github.com/yugabyte/yugabyte-db/pull/17543/commits/7e1ad42a80fe7b575a77cd91b14803bd44ecd0db